docs/pages/dev-guide/developer-guide.md

```diff
@@ -1002,7 +1002,17 @@ plans:
         tasks: [sidecar]
 ```
 
-To initiate this plan, execute an HTTP POST request against the endpoint `/v1/plans/sidecar-example/start`. Its progress can be monitored like any other plan: by issuing GET requests against the `/v1/plans/side-car-example` endpoint.
+Note that the command definition for the sidecar task includes an environment variable, `PLAN_PARAMETER`, that is not defined elsewhere in the service definition. This parameter can be supplied by the user at the time that the plan is initiated, with that parameter being propagated to the environment of every task launched by the plan. To initiate the plan, execute an HTTP POST request against the endpoint `/v1/plans/sidecar-example/start`, with the header `Content-Type: application/json` set and with a JSON body consisting of environment variable name/value pairs as in the following:
```

Comment:
You might want to clarify whether multiple parameters are supported here. From the example, it looks like only one plan parameter is supported. It might be better to use a more realistic example, as the terms "plan_parameter" and "sidecar" may be confusing. How about something like "SIDECAR_MSG": "Sidecar started".